seasmon_xr/accessors.py

```diff
@@ -78,6 +113,60 @@ def _gen_labels(x, lbl, c):
         return f"{x.year}{x.month:02}" + f"{lbl}{int(x.day//c+1)}"
 
 
+class Period(AccessorTimeBase):
```

Comment:
This class is purely meant to serve as parent for the individual periods. However, since it doesn't have the necessary class variables it will lead to an  exception if an instance is created and one of the relevant properties is accessed.

I tried to use abstract base classes with abstract methods to make this clearer, but that didn't work with `xarray`accessors (at least I couldn't manage). 
So please see if OK or if there's a better way.